tests/testthat/test-stan-generated_quantities.R

```diff
@@ -0,0 +1,16 @@
+context("estimate_infections")
+if (!testthat:::on_cran()) {
+  files <- c("pmfs.stan", "infections.stan", "generated_quantities.stan")
+  suppressMessages(expose_stan_fns(files, target_dir = system.file("stan/functions", package = "EpiNow2")))
+}
+
+
+test_that("calculate_growth works as expected", {
+  skip_on_cran()
+  expect_equal(calculate_growth(rep(1, 5), 1), rep(0, 4))
+  expect_equal(round(calculate_growth(1:5, 2), 2), c(0.41, 0.29, 0.22))
+  expect_equal(round(calculate_growth(exp(0.4*1:5), 2), 2), rep(0.4, 3))
+  expect_error(calculate_growth(1:5, 6))
+  expect_error(calculate_growth(1:5, 0))
+})
```

Comment:
Could also add tests to compare the outputs from `calculate_growth` and `R_to_growth`, and to check if there's discontinuities in the forecasts (maybe by comparing the difference between the first forecase time step and the previous day within a reasonable threshold).